test/prequestTests.js

```diff
@@ -36,11 +36,28 @@ describe('Prequest GET tests', function () {
   it('should fail with unknown route', function (done) {
     prequest(url + '/unknown')
       .catch(function (err) {
+        assert(err instanceof Error, 'Err must be an actual error');
+        assert(err.message.indexOf('404') !== -1,
```

Comment:
Perhaps replace the 2 asserts in L40 & L41 with: 
assert.equal(err.message.trimRight(), 'HTTP 404: Cannot GET /unknown')